geomesa-datastore-bundle/geomesa-datastore-processors/pom.xml

```diff
@@ -61,6 +61,11 @@
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-convert-all_2.11</artifactId>
         </dependency>
+        <!-- bring in postgis to make the PutGeoTools processor work with it out of the box -->
+        <dependency>
+            <groupId>org.geotools.jdbc</groupId>
+            <artifactId>gt-jdbc-postgis</artifactId>
+        </dependency>
```

Comment:
Does the bundle need to have the PostGIS JDBC driver mentioned explicitly?  Or does this work out just fine?

Reply:
this seemed to work fine. i think the driver is a  transitive dependency of this artifact?